Fix build failure

NiteshKant opened the pull request:
rxjava-0.18.1 removes all math related methods. Using rxjava-math instead.

Commit:
Fix build failure

rxjava-0.18.1 removes all math related methods. Using rxjava-math instead.

cloudbees-pull-request-builder commented:
[RxNetty-pull-requests #30](https://netflixoss.ci.cloudbees.com/job/RxNetty-pull-requests/30/) FAILURE
Looks like there's a problem with this pull request

Commit:
Fix test failure

New rx-java version thrown NoSuchElementException from last() operator when there is no element.

cloudbees-pull-request-builder commented:
[RxNetty-pull-requests #31](https://netflixoss.ci.cloudbees.com/job/RxNetty-pull-requests/31/) FAILURE
Looks like there's a problem with this pull request

Commit:
Fixing groovy examples build failure

cloudbees-pull-request-builder commented:
[RxNetty-pull-requests #32](https://netflixoss.ci.cloudbees.com/job/RxNetty-pull-requests/32/) SUCCESS
This pull request looks good
